Makefile

```diff
@@ -25,6 +25,12 @@ install:
 	tar -x --xattrs-include=* -f ../$(BASE) -C $(DESTDIR)
 	# ensure resolving works inside the chroot
 	cat /etc/resolv.conf > $(DESTDIR)/etc/resolv.conf
+	# copy-in launchpad's build archive
+	if grep -q ftpmaster.internal /etc/apt/sources.list; then \
+		cp /etc/apt/sources.list $(DESTDIR)/etc/apt/sources.list; \
+		cp /etc/apt/trusted.gpg $(DESTDIR)/etc/apt/ || true; \
```

Comment:
I guess we need the || true here because this file is not always available ?

Reply:
Correct, it depends on target release and which archives are configured.

For example, Ubuntu Archive Bionic+ only ship /etc/apt/trusted.gpg.d/* snippets and don't have any /etc/apt/trusted.gpg

If PPAs are activated, one may have keys in /etc/apt/trusted.gpg in addition to those in /etc/apt/trusted.gpg.d/*